releases/patch-releases.md

```diff
@@ -74,7 +112,8 @@ days after cherry-picks deadline to allow all release-blocking tests to run.
 
 | **Patch Release** | **Cherry-picks deadline** | **Target date** | **Owner** |
 | --- | --- | --- | --- |
-| 1.12.6 | TBA | TBA | TBA |
+| 1.12.7 | TBA | TBA | @feiskyer |
```

Comment:
also update the date for next releases? I think we've agreed to cut a set of patch releases next monday